data/Remnant Cognizance.txt

```diff
@@ -0,0 +1,626 @@
+			`	Chilia notices you eyeing the deviations. "Perceptive, captain. Our sensors have reported some unusual energy signatures coming from this location, and scouts in the area say they are picking up signs of a new system. Unfortunately, they could locate neither a hyperjump route nor a wormhole that would allow them to reach it." He looks up at you. "We are hoping that you might be able to figure out a way in."`
+			choice
+				`	"I could see if I can reach it with a Jump Drive."`
+				`	"I will see if I can find another way."`
+			`	"Thank you, Captain <first>. We have some ships that can follow, but you have more experience with things like this, so we would like you to lead the way."`
+				accept
+	on enter "Postverta"
+		dialog
+			`As the sensors re-establish their depiction of the system around you, an alien station immiediately becomes apparent. It appears to be old and mostly inactive, but your sensors do pick up ocasional bursts of energy. Oddly, your sensors are reporting faint signs of life almost everywhere.`
+	on stopover
+		dialog
+			`The station is surprisingly still active, despite its apparent age. You poke around a bit, but its mysteries remain just that.`
+	on complete
+		conversation
+			`As you arrive back at Viminal you broadcast a warning to the port authorities that you have something following you. You inform them that it isn't hostile, but it is an unknown that originated in the new system.`
+
+
+
+mission "Remnant: Cognizance 12b"
+	landing
+	invisible
+	source "Ssil Vida"
+	destination "Viminal"
+	to offer
+		has "Remnant: Cognizance 12: active"
+	to complete
+		has "Remnant: Cognizance 12: done"
+	npc accompany
+		government "Indigenous Lifeform"
+		personality waiting mute
+		ship "Void Sprite Elder" "Unknown Entity"
+	
+
+
+mission "Remnant: Cognizance 13"
+	name "Follow the Creature"
+	description "The creature seems to be creating constellations and trying to get you to follow it."
+	source "Viminal"
+	waypoint
+		system Peragenor
+		system Convector
+		system Nenia
+	stopover Nasqueron
+	to offer
+		has "Remnant: Cognizance 12: done"
+	on offer
+		conversation
+			`Quite a few Remnant have gathered around the Starport, watching the movements of the strange creature. It gestures wildly in various ways, then freezes in several different positions with the tips of certain tentacles glowing with energy. You note that several anti-air batteries are in position around the area in case the creature becomes violent.`
```

Comment:
`Quite a number of Remnants have gathered around the Spaceport, watching the movements of the strange creature. It gestures wildly in various ways, then freezes in several different positions with the tips of certain tentacles glowing with energy. You note that several anti-air batteries are in position around the area in case the creature becomes violent.`

Reply:
Changed.